server/repository/src/migrations/v1_01_03/mod.rs

```diff
@@ -0,0 +1,45 @@
+use crate::StorageConnection;
+
+use super::{version::Version, Migration};
+
+pub(crate) struct V1_01_03;
+
+impl Migration for V1_01_03 {
+    fn version(&self) -> Version {
+        Version::from_str("1.1.3")
+    }
+
+    #[cfg(feature = "postgres")]
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        use crate::migrations::sql;
+
+        sql!(
+            connection,
+            r#"ALTER TABLE store 
+                ADD logo TEXT;"#
+        )?;
+        Ok(())
+    }
+    #[cfg(not(feature = "postgres"))]
+    fn migrate(&self, _: &StorageConnection) -> anyhow::Result<()> {
+        Ok(())
```

Comment:
Its the same migration for postgres and sqlite so you can remove the `cfg()` macro and only have one `fn migration` method...